scene/gui/box_container.cpp

```diff
@@ -290,6 +288,12 @@ Size2 BoxContainer::get_minimum_size() const {
 	return minimum;
 }
 
+void BoxContainer::_update_theme_item_cache() {
+	Container::_update_theme_item_cache();
+
+	theme_cache.separation = get_theme_constant(SNAME("separation")); //,vertical?"VBoxContainer":"HBoxContainer");
```

Comment:
I guess the dead code comment can be removed.

Reply:
Ah, I only just got what it was for. It's to fetch from a specific type, duh.